cmd/root.go

```diff
@@ -279,6 +282,28 @@ func (r *Root) execute() {
 
 	//Set the version template for version command
 	rootCmd.SetVersionTemplate(`{{with .Version}}{{printf "Keploy %s" .}}{{end}}{{"\n"}}`)
+	currentVersion := utils.Version
+	// Show update message only if it's not a dev version
+	if !strings.HasSuffix(currentVersion, "-dev") {
+		// Check for the latest release version
+		releaseInfo, err := utils.GetLatestGitHubRelease()
+		if err != nil {
+			r.logger.Debug("Failed to fetch the latest release version", zap.Error(err))
+			return
+		}
+		if releaseInfo.TagName != currentVersion {
+			updatetext := models.HighlightGrayString("keploy update")
+			const msg string = `
+               ╭─────────────────────────────────────╮
+               │ New version available:              │
+               │ %v  ---->   %v       │
+               │ Run %v to update         │
+               ╰─────────────────────────────────────╯
+			   `
+			versionmsg := fmt.Sprintf(msg, currentVersion, releaseInfo.TagName, updatetext)
+			fmt.Printf(versionmsg)
```

Comment:
shouldn't we also give an option to the user to update keploy? or atleast tell him how he can do so by mentioning the command here.

Reply:
In my opinion asking the user to update immediately would be a bad user experience. We can however mention the command here to do so.